include/enrico/openmc_heat_driver.h

```diff
@@ -42,9 +44,15 @@ class OpenmcHeatDriver : public CoupledDriver {
   std::unordered_map<int, std::vector<int>> ring_to_cell_inst_;
   std::unordered_map<int, std::vector<int>> cell_inst_to_ring_;
 
+  // Mapping of surrogate fluid elements to OpenMC cell instances and vice versa
+  std::unordered_map<int, std::vector<int>> elem_to_cell_inst_;
```

Comment:
This doesn't need to be a map, a `std::vector` would do. In fact, the one below could also be a `vector`, but in the interest of matching what we do for `OpenmcNekDriver`, maybe just change this one for now.